lib/browser/objects-registry.js

```diff
@@ -49,14 +51,28 @@ class ObjectsRegistry {
   // Dereference an object according to its ID.
   // Note that an object may be double-freed (cleared when page is reloaded, and
   // then garbage collected in old page).
-  remove (webContents, contextId, id) {
+  // rendererSideRefCount is the ref count that the renderer process reported
+  // at time of GC if this is different to the number of references we sent to
+  // the given owner then a GC occurred between a ref being sent and the value
+  // being pulled out of the weak map.
+  // In this case we decrement out ref count and do not delete the stored
+  // object
```

Comment:
Can you add the link of this PR in the comment as it describes why we introduced `rendererSideRefCount`?

```md
For more details on why we introduced rendererSideRefCount, please see https://...
```

Otherwise some other developer may want to optimize this code out a few years later.